dbschema/prompt-variant-response.esdl

```diff
@@ -0,0 +1,34 @@
+module default {
+  type PromptVariantResponse extending Resource {
+    creator: User;
+    multi link responses := VariantResponse;
+  }
+
+  abstract type VariantResponse extending Resource {
+    variant: VariantResponse::Variant;
+    response: RichText;
+    creator: User;
+  }
+  
+  type PromptResponse extending Resource {
+    parent: str;
+    prompt: Prompt;
+    response: RichText;
+  }
+  
+  type Prompt extending Resource {
+    text: RichText;
+    shortLabel: str;
+    min: int16;
+    max: int16; 
+  }
```

Comment:
Currently Prompts only exist in app code. This is one of a few like this.
We should keep it this way at least until after migration.
So delete this.